### PR TITLE
Added automated test to test multitenant --limit flag

### PR DESCRIPTION
**Pull request by shivaji-dgraph**
In this test we are testing the rate limit in multitenant cluster.

-  We created a multi-tenant cluster with a --limit flag set with the uid lease limit of 50 and tried to lease more uids than the lease limit and checks the error message’s

### CODE REVIEW

File: systest/multi-tenancy/basic_test.go [hunk 1]
```diff
@@ -37,6 +37,11 @@ import (
 	"github.com/dgraph-io/dgraph/x"
 )
 
+type inputTriplets_Count struct {
+	lower_limit int
```
Comment (matthewmcneely) [inline systest/multi-tenancy/basic_test.go:41]: IIRC, we're following the Google Go style guide. In which case these should be camelCase and not underscore... https://google.github.io/styleguide/go/guide#mixedcaps

**Review by matthewmcneely — CHANGES_REQUESTED** [review]

File: systest/multi-tenancy/basic_test.go (hunk 1 above)
Comment (shivaji-dgraph) [reply systest/multi-tenancy/basic_test.go]: done @matthewmcneely

File: systest/multi-tenancy/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    ports:
+      - 8080
+      - 9080
+    labels:
+      cluster: test
+      service: alpha
+    command: /gobin/dgraph  ${COVERAGE_OUTPUT} alpha --encryption "key-file=/dgraph-enc/enc-key;" --my=alpha1:7080 --zero=zero1:5080,zero2:5080,zero3:5080 --expose_trace --profile_mode block --block_rate 10 --logtostderr -v=2
+      --security "whitelist=10.0.0.0/8,172.16.0.0/12,192.168.0.0/16;"
+      --acl "secret-file=/dgraph-acl/hmac-secret; access-ttl=20s;"
+
```
Comment (mangalaman93) [inline systest/multi-tenancy/docker-compose.yml:45]: remove these extra lines

File: testutil/multi_tenancy.go [hunk 3]
```diff
@@ -411,3 +411,15 @@ export(input: {destination: $dst, format: $f, accessKey: $acc, secretKey: $sec})
 	require.NoError(t, err)
 	return resp
 }
+
+func AddNumberOfTriples(t *testing.T, dg *dgo.Dgraph, start, end int) (*api.Response, error) {
```
Comment (mangalaman93) [inline testutil/multi_tenancy.go:415]: this function should just be defined in the test code itself. This doesn't seem like a common function

File: systest/multi-tenancy/basic_test.go [hunk 4]
```diff
@@ -565,6 +570,29 @@ func TestTokenExpired(t *testing.T) {
...
+	galaxyToken, err := testutil.Login(t,
+		&testutil.LoginParams{UserID: "groot", Passwd: "password", Namespace: x.GalaxyNamespace})
+	require.NoError(t, err)
+	// Create a new namespace
+	ns, err := testutil.CreateNamespaceWithRetry(t, galaxyToken)
+	require.NoError(t, err)
+	dc := testutil.DgClientWithLogin(t, "groot", "password", ns)
+	require.NoError(t, dc.Alter(context.Background(), &api.Operation{
+		Schema: `name: string .`}))
+	// trying to load more tripletes than allowed. It should through error.
```
Comment (mangalaman93) [inline systest/multi-tenancy/basic_test.go:584]: triplets

File: systest/multi-tenancy/basic_test.go [hunk 5]
```diff
@@ -565,6 +570,29 @@ func TestTokenExpired(t *testing.T) {
...
+	_, err = testutil.AddNumberOfTriples(t, dc, testInputs[0].lowerLimit, testInputs[0].upperLimit)
+	require.Error(t, err)
+	require.Contains(t, err.Error(), "Requested UID lease(53) is greater than allowed(50).")
+	_, err = testutil.AddNumberOfTriples(t, dc, testInputs[1].lowerLimit, testInputs[1].upperLimit)
+	require.NoError(t, err)
+	// we have set uid-lease=50 so we are trying lease more uids. It should through error.
```
Comment (mangalaman93) [inline systest/multi-tenancy/basic_test.go:590]: it should return error

**Review by mangalaman93 — CHANGES_REQUESTED** [review]
minor changes, looks simple enough!

File: testutil/multi_tenancy.go (hunk 3 above)
Comment (shivaji-dgraph) [reply testutil/multi_tenancy.go:415]: @mangalaman93 We can use this function in future test where we need to add number of triplets.

File: systest/multi-tenancy/docker-compose.yml (hunk 2 above)
Comment (shivaji-dgraph) [reply systest/multi-tenancy/docker-compose.yml:43]: done

File: systest/multi-tenancy/basic_test.go (hunk 4 above)
Comment (shivaji-dgraph) [reply systest/multi-tenancy/basic_test.go]: done

File: systest/multi-tenancy/basic_test.go (hunk 5 above)
Comment (shivaji-dgraph) [reply systest/multi-tenancy/basic_test.go]: done

**Review by mangalaman93 — APPROVED** [review]

**Review by anurags92 — APPROVED** [review]

**Review by matthewmcneely — APPROVED** [review]

**Review by mangalaman93 — APPROVED** [review]

**Pull request merged**